video-library/src/components/Trending/TrendingPage/TrendingPage.css

```diff
@@ -0,0 +1,76 @@
+.trending_filter
+
+{
+    display: flex;
+    align-items: center;
+    display: flex;
+    color: #606060;
+    font-size: xx-small im !important;
+}
+
+.trending_filter>h2 
+{
+    margin-left: 5px;
+    font-size: 25px;
+    color: black
+}
+.trending_filter hr 
+{
+    height: 1px;
+    border: 0;
+    background-color: lightgray;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+.avataricontrend
+{
```

Comment:
I have a suggestion, give hyphen or underscore between words to make class name more readable